cli/medperf/utils.py

```diff
@@ -237,23 +243,39 @@ def combine_proc_sp_text(proc: spawn) -> str:
             logging.error("Process timed out")
             raise ExecutionError("Process timed out")
         line = line.decode("utf-8", "ignore")
-        if line:
+
+        if not line:
```

Comment:
I suggest we have the logic of "deciding if we want to filter" be encapsulated in a separate function